src/js/components/TextInput/TextInput.js

```diff
@@ -228,6 +248,9 @@ const TextInput = forwardRef(
       placeholder && typeof placeholder !== 'string' && !value;
 
     let drop;
+    const extraProps = {
+      onSelect: handleTextSelect.current && handleTextSelect.current,
```

Comment:
Why the `&&` for identical conditions?

Reply:
removed, just old logic that was never completely deleted.